rapids_config.cmake

```diff
@@ -11,6 +11,10 @@
 # or implied. See the License for the specific language governing permissions and limitations under
 # the License.
 # =============================================================================
+
+set(rapids-cmake-repo bdice/rapids-cmake)
+set(rapids-cmake-branch cccl-2.7.0-rc2)
+
```

Comment:
This PR will merge after https://github.com/rapidsai/rapids-cmake/pull/710. We must revert these changes before merging this PR.
```suggestion
```